src/libraries/Common/src/Interop/Windows/SChannel/Interop.SECURITY_STATUS.cs

```diff
@@ -51,5 +51,6 @@ internal enum SECURITY_STATUS
         DowngradeDetected = unchecked((int)0x80090350),
         ApplicationProtocolMismatch = unchecked((int)0x80090367),
         NoRenegotiation = unchecked((int)0x00090360),
+        KeysetDoesNotExist = unchecked((int)0x80090016)
```

Comment:
It feels like this should be `KeySetDesNotExist` to follow https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509keystorageflags?view=net-6.0 capitalization. 

I'm not 100% this matches the other usage. cc: @bartonjs for second voice.